src/transaction.ts

```diff
@@ -59,6 +60,11 @@ export interface RequestOptions {
   maxResumeRetries?: number;
 }
 
+export interface CommitOptions {
```

Comment:
Thanks so much for the idea. That actually ended up working out so hopefully now it's no longer a breaking change.

Reply:
While the suggestion alleviates the breaking change issue, I think it is worth  noting that it also diverges from the accepted pattern here and in other APIs (I think). I do not believe we are extending `CallOptions` anywhere else. Usually `gaxOptions` is one of the `options`'s properties.

Reply:
Ok, that's reasonable. I also didn't see any other places where we extended `CallOptions`. I'm going to follow-up with the team to see whether we will have a breaking change in other languages.

Reply:
This will likely be a breaking change in a few of the libraries so it may have to be the same in Node.js :( At least, most users won't be setting gaxOptions so it's unlikely that they will have to make a code change.

Reply:
Is it possible the setting to return stats could be set higher up the chain somehow, as opposed to on a per-call basis?

Reply:
Actually, I hadn't thought of exploring that option. Good idea, let me talk to the team that's working on this feature to confirm whether this is acceptable.

Reply:
On second thought, we're adding further commit options very soon (e.g. tags and RPC priority) which needs to be plumbed through to the commit request as is done here, and these are specifically for requests. So setting it at the Table level won't be sufficient :(

Reply:
I made the change based on the recommendation that @bcoe made. I'm still not convinced that this is the way to implement it. I checked the commit history and we only introduced gaxOptions here 5 months back in #1050. So it's extremely unlikely that anyone is setting this option.

Reply:
All sorted, we decided to go ahead with the breaking change given that the impact is minimal and it also means that we won't have to refactor this to create an even bigger breaking change in the future.